src/borg/archiver.py

```diff
@@ -2202,11 +2205,22 @@ def _setup_implied_logging(self, args):
             if args.get(option, False):
                 logging.getLogger(logger_name).setLevel('INFO')
 
+    def _setup_topic_debugging(self, args):
+        """Turn on DEBUG level logging for specified --debug-topics."""
+        debug_topics = {
+            'file-compression': 'borg.debug.file-compression',
+        }
```

Comment:
ok, if you wan to control any logger, couldn't this be just:
```
if '.' not in topic:
    topic = 'borg.debug.' + topic
```
That way we would not have to expand the map with an ever growing amount of known debug loggers.